Lesson 9

p4s1k opened the pull request.

p4s1k committed:
Added Top Selection block.

p4s1k committed:
Changed Png sprite to Svg sprite.
Refactoring code by BEM methodology.

p4s1k committed:
made form like a design

p4s1k committed:
made "what do we offer" and "footer" blocks responsive

p4s1k committed:
made "top-selection" block responsive

AverFrenzy commented on index.html:
```diff
@@ -4,69 +4,352 @@
+            </div>
+          </div>
+          <div class="container top-selection-content__container">
+            <div class="top-selection__heading">
```
Это h1

AverFrenzy commented on index.html:
```diff
@@ -4,69 +4,352 @@
+                  <li>Attractions</li>
+                </ul>
+                <div class="header__menu-icons">
+                  <svg class="header__icon-theme">
```
обвурнуть все эти svg в button т.к в будущем скорее всего это должны быть кнопки и сделать в стилях cursor:pointer;

AverFrenzy commented on index.html:
```diff
@@ -4,69 +4,352 @@
+              <div class="header__menu-row">
+                <ul class="menu-list header__menu-list">
+                  <li>
+                    Stays
```
stays обвернуть в тег а и вторую ссылку тоже

AverFrenzy commented on index.html:
```diff
@@ -4,69 +4,352 @@
+                  </button>
+                </div>
+              </form>
+              <form action="#" class="form">
```
написать что эта форма используется для мобилки в классах,используя БЭМ

AverFrenzy commented on index.html:
```diff
@@ -4,69 +4,352 @@
-                <li>FAQ</li>
-              </ul>
-            </div>
+        <div class="container footer__container">
```
можешь сделать тут вместо div тег footer,так семантически вернее будет

AverFrenzy commented on index.html:
```diff
@@ -4,69 +4,352 @@
+    <div class="top-selection">
+      <div class="wrapper top-selection__wrapper">
+        <div class="container top-selection__container">
+          <div class="header">
```
тут можно вместо div подставить header,так семантически вернее будет

p4s1k committed:
combined two forms into one
changed div's to header, h1, footer.
wrapped header links in a tag
wrapped svg's in button tag
set 'cursor: pointer' to svg buttons and header links
